.github/workflows/check_todos.yml

```diff
@@ -11,7 +11,8 @@ on:
     branches:
       - main
 jobs:
-  Enforce issue references:
+  enforce-issue-references:
+    name: Enforce issue references
```

Comment:
@kostmo you can see the error here:
- https://github.com/swarm-game/swarm/actions/workflows/check_todos.yml

If you use VS Code, the YAML extension should pick up the JSON Schema and show the error immediately:
|![image](https://user-images.githubusercontent.com/44544735/219866882-a484a440-7a23-4ead-8ca2-e4fedb8e7224.png)|
|---|

Reply:
Cool, thanks for fixing.